### PR TITLE
Add a build step to initial setup

### PR DESCRIPTION
**Pull request by gpsamson**
Adds a `yarn build` step to the setup steps in the README. This is temporary additional step until we can debug the possible regression.

## Testing

_Include any additional information about the testing you have completed to
ensure your changes behave as expected. For a speedy review, please check
any of the tasks you completed below during your testing._

- [ ] Added [unit tests](https://github.com/segmentio/action-destinations/blob/main/docs/testing.md#local-end-to-end-testing) for new functionality
- [ ] Tested end-to-end using the [local server](https://github.com/segmentio/action-destinations/blob/main/docs/testing.md#local-end-to-end-testing)
- [ ] [Segmenters] Tested in the staging environment

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -40,6 +40,7 @@ yarn login
 # Requires node 14.17, optionally: nvm use 14.17
 yarn --ignore-optional
 yarn bootstrap
+yarn build
```
Comment (mshwery) [inline README.md:43]: I think we must be missing some ts-jest references somewhere...

Comment (gpsamson) [reply README.md:43]: Let me give it a look

Comment (gpsamson) [reply README.md:43]: Can confirm `ts-jest` is configured for all packages (particularly for destination-actions): https://github.com/segmentio/action-destinations/blob/main/packages/destination-actions/package.json#L47-L56

**Review by mshwery — APPROVED** [review]

**Comment by gpsamson** [comment]
Closing in favor of https://github.com/segmentio/action-destinations/pull/340

**Pull request closed without merging**